app/_locales/en/messages.json

```diff
@@ -401,6 +401,10 @@
   "amount": {
     "message": "Amount"
   },
+  "and": {
```

Comment:
Can we do this with variables in the text instead? I feel like not all languages may work with a simple "and".

Reply:
I do not understand clearly what do you mean. Can you provide an example?

Reply:
This is only used for the last element in the function `constructWarningElementComponentArray`. If some other languages does not use something like `and`, it can be just `,` in translation 🤔

Reply:
Refactored with new translations and new logic for constructing warning elements array. It's needed to handle all the cases with arbitraty number of elements.